README.Rmd

```diff
@@ -27,7 +27,12 @@ knitr::opts_chunk$set(
 )
 ```
 
--   [Randomer Forest](#randomer-forest)
+[![CRAN Status Badge](https://www.r-pkg.org/badges/version/rerf)](https://cran.r-project.org/package=rerf)
+[![arXiv shield](https://img.shields.io/badge/arXiv-1506.03410-red.svg?style=flat)](https://arxiv.org/abs/1506.03410)
+[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.1933344.svg)](https://doi.org/10.5281/zenodo.1933344)
+
+
+-   README
```

Comment:
Can we have this be "Randomer Forest" and have it link there?  it looks weird in the TOC without a link